mkosi

```diff
@@ -2455,9 +2456,9 @@ def install_arch(args: CommandLineArguments, root: str, do_run_build_script: boo
         conf = ["--config", pacman_conf]
 
         try:
-            run(["pacman-key", *conf, "--init"], check=True)
-            run(["pacman-key", *conf, "--populate"], check=True)
-            run(["pacman", *conf, "--noconfirm", "-Sy", *packages], check=True)
+            run(["pacman-key", *conf, "--init"])
+            run(["pacman-key", *conf, "--populate"])
+            run(["pacman", *conf, "--noconfirm", "-Sy", *packages])
         finally:
             # Kill the gpg-agent started by pacman and pacman-key.
             run(['gpgconf', '--homedir', os.path.join(root, 'etc/pacman.d/gnupg'), '--kill', 'all'])
```

Comment:
Same here, CI works so I guess we can check the result here safely as well?

Reply:
If those fail, we'd need to die anyway, something the code would have already done, so this is fine.